server/manifest.go

```diff
@@ -25,13 +25,35 @@ const manifestStr = `
     },
     "executable": ""
   },
-  "webapp": {
-    "bundle_path": "webapp/dist/main.js"
-  },
   "settings_schema": {
     "header": "",
     "footer": "",
-    "settings": []
+    "settings": [
+      {
+        "key": "OAuthClientID",
+        "display_name": "OAuth Client ID:",
+        "type": "text",
+        "help_text": "The client ID for the OAuth app registered with Twitter.",
+        "placeholder": "",
+        "default": "WkEzuWVPGHSmO4vm0yvdF8bq4"
+      },
+      {
+        "key": "OAuthClientSecret",
+        "display_name": "OAuth Client Secret:",
+        "type": "text",
+        "help_text": "The client secret for the OAuth app registered with Twitter.",
+        "placeholder": "",
+        "default": "LwAgBVckO0EQlI2zka55ZFAN16xMRY8T0zvzVI4KbVlCK6tBN1"
+      },
+      {
+        "key": "EncryptionKey",
```

Comment:
Are we using this? In theory, there is no need to encrypt the tokens, so you can get rid of this and the logic around the encryption.